### ISSUE TITLE
Wrong default permissions at /var/www/db 

### ISSUE THREAD

**Issue by IlyaSemenov** [robbertkl/docker-roundcube#1]
If this image is started without a linked data container it prints the following error message:

> DATABASE ERROR: CONNECTION FAILED!
> 
> Unable to connect to the database!
> Please contact your server-administrator.

The Dockerfile must run `chown www-data /var/www/db` to fix this problem.

**Comment by robbertkl** [comment]
Hi @IlyaSemenov, the Dockerfile contains `chmod a+rw db`, which should prevent that issue. I've just started a container without any volume binds, and I did not get the same issue.

Can you send me the full `docker run` command you're using?

**Comment by robbertkl** [comment]
Closing this because of inactivity. Please reopen if you're still experiencing the issue, be sure to include your `docker run` command, and perhaps the output of `docker info`.

**Comment by DEV-SOUTH** [comment]
I think I left a wrong note for myself the other day. Indeed, it works well without a linked data container.

The problem appears when you run it this way:

    docker run --rm -it -p 8001:80 -v /srv/roundcube:/var/www/db robbertkl/roundcube

if `/srv/roundcube` didn't exist prior to running the container.

Typically, docker containers prepare their data directories on startup. For instance, standard `redis`, `mysql`, `postgres` and `mongodb` all work fine if you run them linking to non-existent directories. This is not the case with `robbertkl/roundcube`.

**Comment by robbertkl** [comment]
Thanks for the additional info. I'll reopen this and bake a fix soon.

**Comment by IlyaSemenov** [comment]
_Sorry, I posted the comment under the company account by mistake. Repeating it below for the record—_

I think I left a wrong note for myself the other day. Indeed, it works well without a linked data container.

The problem appears when you run it this way:

```
docker run --rm -it -p 8001:80 -v /srv/roundcube:/var/www/db robbertkl/roundcube
```

if `/srv/roundcube` didn't exist prior to running the container.

Typically, docker containers prepare their data directories on startup. For instance, standard `redis`, `mysql`, `postgres` and `mongodb` all work fine if you run them linked to non-existent directories. This is not the case with `robbertkl/roundcube`.

**Comment by robbertkl** [comment]
Just committed a fix, a3139ae. New image is on Docker hub. Let me know if it works for you!
